ssr/src/utils/event_streaming/mod.rs

```diff
@@ -94,3 +96,13 @@ pub async fn stream_to_offchain_agent(event: String, params: String) -> Result<(
 
     Ok(())
 }
+
+#[cfg(clippy)]
```

Comment:
can you give a bit of context for adding these cfg flags?

Reply:
<img width="898" alt="Screenshot 2024-08-22 at 4 44 24 PM" src="https://github.com/user-attachments/assets/2a7e1101-8b93-4b57-93b1-6b66edefb1bd">

Getting clippy errors after adding two tonic dependencies with different features